doc/make.py

```diff
@@ -288,6 +288,13 @@ def zip_html(self):
         os.chdir(dirname)
         self._run_os("zip", zip_fname, "-r", "-q", *fnames)
 
+    def linkcheck(self):
+        """
+        Check for broken links in the documentation.
+        """
+        ret_code = self._sphinx_build("linkcheck")
+        return ret_code
```

Comment:
Nit: if we don't use the variable for anything else, we can eliminate it:

```suggestion
        return self._sphinx_build("linkcheck")
```